src/Rings/mpoly-ideals.jl

```diff
@@ -2102,9 +2103,12 @@ function grassmann_pluecker_ideal(ring::MPolyRing,
     @assert is_integral(c)
     return coeff_ring(numerator(c))
   end
-  h = hom(base_ring(I), ring, coeffmap, gens(ring))
-  converted_generators = elem_type(ring)[h(g) for g in groebner_basis(I; ordering = degrevlex(base_ring(I)))]
-  ideal(IdealGens(ring, converted_generators, o;
+  if !is_graded(ring)
```

Comment:
you need to make sure in both cases gr_ring is defined
````suggestion`
  gr_ring = is_graded(ring) ? ring : grade(ring)
```